crates/biome_cli/tests/snapshots/main_cases_included_files/does_handle_only_included_files.snap

```diff
@@ -26,7 +26,7 @@ statement();
 # Emitted Messages
 
 ```block
-Formatted 1 file(s) in <TIME>
+Formatted 1 file in <TIME>. fixed file.
```

Comment:
```suggestion
Formatted 1 file in <TIME>. Fixed file.
```

Reply:
That's a bug right?

Reply:
Yeah, the number of files is missing and t`fixed` is not capitalized.

Reply:
Cheers, that should be fixed now